docs/install/installation/linux_install.md

```diff
@@ -31,8 +31,8 @@ In this video, you will learn how to install and run LibreChat, using Docker on
 - Install Docker: `sudo apt install docker.io && apt install docker-compose -y`
 - Enter the folder: `cd LibreChat`
 - Create the .env file: `cp .env.example .env`
-- Build the Docker image: `docker compose build`
-- Start LibreChat: `docker compose up -d`
+- Build the Docker image: `docker-compose build`
```

Comment:
Yeah, it's probably distro dependent then. The original commands didn't work for me on my WSL Ubuntu 22.04, and also not on my AWS EC2 Ubuntu 22.04.
Perhaps there could be a note or separate section for Ubuntu users but otherwise I guess they could figure it out with chatgpt.
Thanks for checking!

Reply:
@BernardVanTonder Oh I see, I got this guide confused with the one here: https://docs.librechat.ai/deployment/docker_ubuntu_deploy.html#5-install-the-latest-version-of-docker-compose

In the docker ubuntu deployment guide, it specifies how to update docker compose as linked above. Ubuntu 22.04 anywhere is compatible with docker compose v2, it's a matter of adding those notes here as an optional step, and making sure that `docker compose` is preferred if you update to latest compose, but `docker-compose` is necessary if not. The distro may only point to the v1 packages by default.